### ISSUE TITLE
Endpoint with express server very slow

### ISSUE THREAD

**Issue by bergersens** [Josh-McFarlin/remix-image#29]
### Describe the bug

if i use i with the express app setup, the resizer api is extremely slow.

my api endpoint:

```ts
import type { LoaderFunction } from "@remix-run/node";
import { imageLoader, MemoryCache } from "remix-image/serverPure";
import { sharpTransformer } from "~/lib/image-transformer";

export const loader: LoaderFunction = ({ request,params }) => {
  return imageLoader(
   {
      selfUrl: "http://localhost:8080",
      cache: new MemoryCache(),
      defaultOptions: { quality: 75 },
      transformer: sharpTransformer
    },
    request
  )};
```

my express server (./server.js):

```js
const path = require("path");
const express = require("express");
const compression = require("compression");
const morgan = require("morgan");
const { createRequestHandler } = require("@remix-run/express");

const BUILD_DIR = path.join(process.cwd(), "build");

const app = express();

app.use(compression());

app.disable("x-powered-by");

app.use(
  "/build",
  express.static("public/build", { immutable: true, maxAge: "1y" })
);

app.use(express.static("public", { maxAge: "1h" }));

app.use(morgan("tiny"));

app.all(
  "*",
  process.env.NODE_ENV === "development"
    ? (req, res, next) => {
        purgeRequireCache();

        return createRequestHandler({
          build: require(BUILD_DIR),
          mode: process.env.NODE_ENV,
        })(req, res, next);
      }
    : createRequestHandler({
        build: require(BUILD_DIR),
        mode: process.env.NODE_ENV,
      })
);

const port = parseInt(process.env.PORT) || 8080;
app.listen(port, () => {
  console.log(`app listening on port ${port}`);
});

function purgeRequireCache() {
  // purge require cache on requests for "server side HMR" this won't let
  // you have in-memory objects between requests in development,
  // alternatively you can set up nodemon/pm2-dev to restart the server on
  // file changes, but then you'll have to reconnect to databases/etc on each
  // change. We prefer the DX of this, so we've included it for you by default
  for (let key in require.cache) {
    if (key.startsWith(BUILD_DIR)) {
      delete require.cache[key];
    }
  }
}
```

in my live/prod env it is so slow that the server (google cloud run) responded with 503. in my dev env I wait for  an image like 10 seconds 

### Expected behavior

see bug description

### Platform

Chrome 100

**Comment by Josh-McFarlin** [comment]
Hi @bergersens,

I believe this may be related to issue #25, and something is going wrong with `MemoryCache` from some changes that must have occurred in Remix since the last major release of this package. You can see if this is the case by removing `MemoryCache` from your config, but this means each new client will be requesting a brand new image. Unfortunately, I am too busy to begin looking into `MemoryCache` for the next week or so. In the meantime, you can try implementing the first change I mentioned in [this comment](https://github.com/Josh-McFarlin/remix-image/issues/25#issuecomment-1105489887), since it sounds like the issue is related to running the loader and not initializing it.

**Comment by Josh-McFarlin** [comment]
@bergersens Today I released v1.0.0 of Remix-Image and made some changes to MemoryCache and tested the package on different environments, and it appears to be working correctly. Would you mind updating to the latest release to check if the bug still occurs in your environment?
